README.md

```diff
@@ -1,14 +1,14 @@
 # datagrepper
 
-Datagrepper is a web application and JSON API to retrieve historical messages sent via Fedora Messaging. [Datanommer](https://github.com/fedora-infra/datanommer/) is a seperate project and service that consumes messages from the Fedora Messaging queue and puts them in a database. These messages is what datagrepper queries. 
+Datagrepper is a web application and JSON API to retrieve historical messages sent via Fedora Messaging. [Datanommer](https://github.com/fedora-infra/datanommer/) is a separate project and service that consumes messages from the Fedora Messaging queue and puts them in a database. These messages are what datagrepper queries. 
 
-Datagrepper is curently running in production at https://apps.fedoraproject.org/datagrepper/
+Datagrepper is currently running in production at https://apps.fedoraproject.org/datagrepper/
 
 ## Development Environment
 
 Vagrant allows contributors to get quickly up and running with a datagrepper development environment by automatically configuring a virtual machine. 
 
-The datagrepper Vagrant environment configures configures and enables a datanommer service and database. The datanommer instance is configured to be empty when first provisioned, but to consume messages from the stage Fedora Messaging queue.
+The datagrepper Vagrant environment configures and enables a datanommer service and database. The datanommer instance is configured to be empty when first provisioned, but it will consume messages from the staging Fedora Messaging queue.
```

Comment:
```suggestion
The Datagrepper Vagrant environment configures and enables a Datanommer service and database. The Datanommer instance is configured to be empty when first provisioned, but it will consume messages from the staging Fedora Messaging queue.
```